src/WinAppCommunity.Sdk.csproj

```
@@ -5,7 +5,7 @@
 		<LangVersion>12.0</LangVersion>
 		<WarningsAsErrors>nullable</WarningsAsErrors>
 		<GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-		<PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+		<PackageRequireLicenseAcceptance>True</PackageRequireLicenseAcceptance>
```

Let's revert this, the MIT license is very permissive so there's no need to require license acceptance. 
```suggestion
		<PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
```